go/porcelain/deploy.go

```
@@ -820,70 +962,83 @@ func addInternalFilesToDeploy(dir, internalPath string, files *deployFiles, obse
 
-func (l *lazyTempDir) get() (string, error) {
+func (l *lazyTempDir) get() (string, *os.Root, error) {
 	if !l.created {
 		path, err := os.MkdirTemp(l.root, "netlify-deploy-functions-")
 		if err != nil {
-			return "", err
+			return "", nil, err
+		}
+		handle, err := os.OpenRoot(path)
+		if err != nil {
+			os.RemoveAll(path)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)